scrapers/ManyVids.yml

```diff
@@ -19,4 +29,69 @@ xPathScrapers:
         Name: $details//h4/a/text()
       Image: //div[@id="rmpPlayer"]/@data-video-screenshot
 
-# Last Updated May 15, 2020
+  performerSearch:
+    performer:
+      Name: //*[contains(concat(' ', normalize-space(@class), ' '), ' profile-pic-name ')]/a/text()
+      URL:
+        selector: //*[contains(concat(' ', normalize-space(@class), ' '), ' profile-pic-name ')]/a/@href
+        replace:
+          - regex: '/(?:Store/Videos/)?$'
+            with: /About/
+
+  performerScraper:
+    common:
+      $aboutlabel: //*[@class='mv-about__container__details__list-unit']/span[@class='mv-about__container__details__list-label']
+    performer:
+      Name: //*[@class='mv-about__banner-name ']/text()
+      Gender:
+        selector: $aboutlabel[contains(text(), 'Gender')]/following-sibling::*
+        replace:
+          - regex: Trans
+            with: transgender_female
+      URL: //a[@data-tab='about']/@href
+      Twitter: //*[@class='mv-about__banner-social']/a/i[@class='icon twitter-icon']/../@href
+      Instagram: //*[@class='mv-about__banner-social']/a/i[@class='icon instagram-icon']/../@href
+      Birthdate:
+        selector: $aboutlabel[contains(text(), 'Age')]/following-sibling::*
+        replace:
+          - regex: ' \(.*\)$'
+            with: ''
+        parseDate: 2006.01.02
+      Ethnicity:
+        selector: $aboutlabel[contains(text(), 'Ethnicity')]/following-sibling::*
+      Country:
+        selector: $aboutlabel[contains(text(), 'Nationality')]/following-sibling::*
```

Comment:
from a quick test, since we are looking for the country
`selector: $aboutlabel[contains(text(), 'Nationality')]/following-sibling::*/img/@alt` 
seems to fetch what we need and in most cases spares us from editing

Reply:
Nice, I didn't even catch the alt on the img